### DISCUSSION TITLE
Theme header styles completely broken with rgba colors

### DISCUSSION THREAD

**Discussion by iwasrobbed** [glideapps/glide-data-grid#612] [Q&A]
### Are RGBA colors not officially supported for themes?

This is what the header looks like with `bgHeader`, `bgHeaderHasFocus`, `bgHeaderHovered` all set to the same exact color values (`rgba(255, 255, 255, 0.08)`)

<img width="1243" alt="Screen Shot 2023-01-20 at 1 06 36 PM" src="https://user-images.githubusercontent.com/317671/213774496-c66404c2-f43f-4952-a183-7172e0a939b0.png">

1. After hovering over, it seems to ignore the theme colors and paints it white (e.g. it applies the rgba color over top itself and blends the two until it turns whiter)
2. When focusing on a given cell, it paints all header cells a lighter variation of the original header color
3. When you tap on a header to select it, it changes it to the accent color
4. Then when you scroll horizontally, it does this... just completely breaks and blurs all header text togther

<img width="1241" alt="Screen Shot 2023-01-20 at 1 00 10 PM" src="https://user-images.githubusercontent.com/317671/213774154-1fa45e79-28de-4345-99b1-982283fd3137.png">

**Comment by jassmith** [comment]
oh uh, yeah definitely can't put rgba there. I will force it to pre-blend the color with a solid background. The canvas it paints to is RGB only.

**Reply by jassmith** [reply]
Not saying that rgba shouldn't work, just that I need to check its rgba and pre-blend it with an rgb if it is
